src/stack_check.h

```diff
@@ -40,14 +44,19 @@ static void stack_check_init(void)
 	#undef EXPORT_PLUGSCHED
 	#undef EXPORT_CALLBACK
 
+	kallsyms_lookup_size_offset((unsigned long)__orig___schedule,
+				&orig_schedule_size, NULL);
+	kallsyms_lookup_size_offset((unsigned long)__mod___schedule,
+				&mod_schedule_size, NULL);
+
```

Comment:
Here, orig_schedule_size and mod_schedule_size variables can be reused the orig___schedule_size and mod___schedule_size defined head_jump.h which need not get the value again.